golem_messages/message/tasks.py

```diff
@@ -96,6 +96,7 @@ class TaskToCompute(TaskMessageMixin, base.Message):
         'compute_task_def',
         'package_hash',
         'concent_enabled',
+        'subtask_price', # total subtask price computed as price*subtask_timesout
```

Comment:
typo -> should be `subtask_timeout` + I'd call it just `price`

Reply:
@shadeofblue, Łukasz wrote on issue 
`And a proper comment like "total subtask price computed as price*subtask_timesout". And maybe some validation too, like subtask_price > 0.`
Shouldn't be `subtask_price` as he said?

Reply:
@kbeker nah, just make it `price` so as not to cause further proliferation of unneeded `subtask_` vs `task_` prefixes...

Reply:
thank you